src/navbar/drawer.styles.tsx

```diff
@@ -102,3 +102,48 @@ export const CloseButton = styled(IconButton)`
         color: ${Color.Primary};
     }
 `;
+
+export const NavBrandContainer = styled.div`
+    display: flex;
+    height: 100%;
+    flex-direction: row;
+    align-items: center;
+    position: relative;
+`;
+
+export const NavLogoContainer = styled.div`
+    display: flex;
+    height: 100%;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    position: relative;
+`;
+export const NavSeprator = styled.div`
+    display: flex;
+    background-color: ${Color.Neutral[5]};
+    height: 2.8125rem;
```

Comment:
This is an odd value. 2.8125 translates to 45px. Most of the measurements runs in multiples of 4. Could you clarify the correct value instead?